contentgrid-spring-data-rest/src/test/java/com/contentgrid/spring/data/rest/hal/forms/ContentGridHalFormsConfigurationTest.java

```diff
@@ -170,9 +219,7 @@ void updateFormFieldForConstrainedAttributeOptions() throws Exception {
                                             name: "gender",
                                             type: "radio",
                                             options: {
-                                                inline: [ "female", "male" ],
-                                                minItems: 0,
-                                                maxItems: 1
```

Comment:
Why was this removed?

Reply:
The json payload for this test still contains minItems: 0, maxItems: 1, it's now just only tested in the search requests.

Reply:
I think this was a rebase error; I pulled one commit out into 2 PRs. There is no reason why this shouldn't be tested.